react/IconButton/Readme.md

```diff
@@ -1,30 +1,58 @@
 A component suitable to be used when an Icon should be used as a button.
-Provides hover, active styles + accessible size (48px).
+Provides hover, active styles + accessible size.
 
 ```jsx
 import IconButton from 'cozy-ui/transpiled/react/IconButton'
 import Icon from 'cozy-ui/transpiled/react/Icon'
 
+// <-- only useful for the documentation
 import TrashIcon from 'cozy-ui/transpiled/react/Icons/Trash'
-import LeftIcon from 'cozy-ui/transpiled/react/Icons/Left'
 import RestoreIcon from 'cozy-ui/transpiled/react/Icons/Restore'
-import RightIcon from 'cozy-ui/transpiled/react/Icons/Right'
 import CrossIcon from 'cozy-ui/transpiled/react/Icons/Cross'
+import Grid from 'cozy-ui/transpiled/react/MuiCozyTheme/Grid'
+import Stack from 'cozy-ui/transpiled/react/Stack'
+import Typography from 'cozy-ui/transpiled/react/Typography'
+import Variants from 'cozy-ui/docs/components/Variants'
+
+const columns = [{ title: 'default', disabled: false }, { title: 'disabled', disabled: true }]
+const initialVariants = [{ small: false, medium: true, large: false }]
```

Comment:
everything is ok for me.

In the styleguidist, I don't see any difference between small and medium button, is it normal?

Reply:
arf on s'est croisé ! Exact, non ce n'est pas normal, je le corrige dans la PR qui suit => https://github.com/cozy/cozy-ui/pull/2121